go.mod

```diff
@@ -9,92 +9,231 @@ module k8s.io/kubernetes
 go 1.19
 
 require (
+	bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898
```

Comment:
hmm... I didn't expect any of these changes, only the golang.org/x/net one... can you make sure you're running `pin-dependency.sh` and `update-vendor.sh` with go1.19+ and are on the latest release-1.23 branch?